source/community.applications/usr/local/emhttp/plugins/community.applications/scripts/updatePLG.sh

```
@@ -1,29 +1,10 @@
-
-if [ -z "${1:-}" ]; then
-	echo "Usage: $0 <plugin-file-name>" >&2
-	exit 2
-fi
 
 echo Getting update information...
 /usr/local/emhttp/plugins/dynamix.plugin.manager/scripts/plugin check "$1" > /dev/null 2>&1
-UPDATEVER=$(/usr/local/emhttp/plugins/dynamix.plugin.manager/scripts/plugin version "/tmp/plugins/$1")
-NEWVER=$(/usr/local/emhttp/plugins/dynamix.plugin.manager/scripts/plugin version "/var/log/plugins/$1")
-if [ "$UPDATEVER" = "$NEWVER" ]; then
+UPDATEVER=$(/usr/local/emhttp/plugins/dynamix.plugin.manager/scripts/plugin version /tmp/plugins/$1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)